python/cuspatial/cuspatial/core/spatial/distance.py

```diff
@@ -213,3 +221,128 @@ def pairwise_linestring_distance(offsets1, xs1, ys1, offsets2, xs2, ys2):
             )
         }
     )
+
+
+def pairwise_point_linestring_distance(
+    points: GeoSeries, linestrings: GeoSeries
+):
+    """Compute shortest distance between pairs of points and linestrings
```

Comment:
I think the body of this docstring needs something explaining that this is pairwise, not all-pairs -- it's not clear from the input parameters how they are paired up.

Reply:
Done